schemas/+acq/checkFileversion.m

```diff
@@ -0,0 +1,18 @@
+
+function oldversion=checkFileVersion(fn)
+
+oldversion = true ;
+try
```

Comment:
it works if the file name is /Volumes/M....
Is this what you meant in the comment getFile... To translate the file name for the platform used ?